projects/vdk-plugins/vdk-impala/.plugin-ci.yml

```diff
@@ -4,15 +4,9 @@
 image: "python:3.7"
 
 .build-vdk-impala:
-  image: docker:19.03.8
-  services:
-    - docker:19.03.8-dind
   variables:
-    DOCKER_HOST: tcp://localhost:2375
-    DOCKER_DRIVER: overlay2
-    DOCKER_TLS_CERTDIR: ""
     PLUGIN_NAME: vdk-impala
-  extends: .build-plugin
+  extends: .build-plugin-dind
```

Comment:
what does `dind` stand for here?

Reply:
Docker in Docker (also known as dind) is, as the name implies, running Docker on top of a Docker container. We are starting docker containers inside a docker container. 

All Gitlab runners and jobs are docker containers (Kubernetes pods really) .